### ISSUE TITLE
Migration count incorrect

### ISSUE THREAD

**Issue by neilstevenson** [hazelcast/hazelcast#19552]
Using Hazelcast Enterprise 5.0-SNAPSHOT (20210915 - e0793e4, 9ae3a4b)

```
10:49:43.690 [hz.heuristic_matsumoto.migration] INFO  com.hazelcast.internal.partition.impl.MigrationManager - [10.172.4.22]:5701 [green] [5.0-SNAPSHOT] All migration tasks have been completed. (repartitionTime=Thu Sep 16 10:49:43 UTC 2021, plannedMigrations=271, completedMigrations=271, remainingMigrations=0, totalCompletedMigrations=281
```

281 is an incorrect total

Related perhaps to https://github.com/hazelcast/hazelcast/issues/19551

**Comment by mmedenjak** [comment]
I think the total completed migrations does not need to be equal to the partition count and may include migrations that simply move a replica up/down the partition ownership array. @vbekiaris @fbarotov can you confirm?

**Comment by fbarotov** [comment]
Yes, (total) completed migrations does not have to equal partitions count, since some partitions might not migrate at all, although replica up/down ownership movement is considered as a migration. Regarding `completedMigrations` count, it is for one phase of repartitioning, whereas `totalCompletedMigrations` is total migration count since HZ instance has been alive, so it is always `totalCompletedMigrations >= completedMigrations `
